extensions/intellij/intellij_generator_plugin/src/main/resources/templates/rx_bloc_with_default_states/rx_bloc.dart.template

```diff
@@ -12,19 +12,19 @@ abstract class ${bloc_pascal_case}BlocEvents {
 /// A contract class containing all states of the ${bloc_pascal_case}BloC.
 abstract class ${bloc_pascal_case}BlocStates {
   /// The loading state
-  Stream<bool> get isLoading;
+  Stream<bool>${bloc_use_null_safety} get isLoading;
```

Comment:
@nikolayIvanovPrime, could you please explain why isLoading needs to be nullable?